DOCUMENTATION.md

```diff
@@ -1,62 +1,67 @@
+        * [downlink_callback](#downlink_callback)
+    * [set_close_callback](#set_close_callback)
+        * [close_callback](#close_callback)
+    * [send](#send)
+    * [UplinkMessage](#uplinkmessage)
 * [ApplicationClient](#applicationclient)
-  * [get](#get)
-  * [set_payload_format](#set_payload_format)
-  * [set_custom_payload_functions](#set_custom_payload_functions)
-  * [unregister](#unregister)
-  * [device](#device)
-  * [devices](#devices)
-  * [update_device](#update_device)
-  * [delete_device](#delete_device)
-  * [DeviceObject](#deviceobject)
-  * [Application](#application)
+    * [get](#get)
+    * [set_payload_format](#set_payload_format)
+    * [set_custom_payload_functions](#set_custom_payload_functions)
+    * [unregister](#unregister)
+    * [device](#device)
+    * [devices](#devices)
+    * [update_device](#update_device)
+    * [delete_device](#delete_device)
+    * [Device](#deviceobject)
+    * [Application](#application)
 * [HandlerClient](#handlerclient)
-  * [data](#data)
-  * [application](#application)
+    * [data](#data)
+    * [application](#application)
 * [Troubleshooting](#troubleshooting)
 
-
 ## Description
-This package provides you an easy way to exchange traffic with The Things Network via MQTT and manage your applications.
 
+This package provides you an easy way to exchange traffic with The Things Network via MQTT and manage your applications.
 
 ## MQTTClient
+
 The class constructor can be called following this scheme:
+
 ```python
-MQTTClient(app_id, app_access_key, **kwargs)
+MQTTClient(app_id, app_access_key, mqtt_address="", discovery_address="discovery.thethings.network:1900")
 ```
-- `app_id`: **string**  this the name given to the application when it was created.
-![Screenshot of the console with app section](./images/app-console.png?raw=true)
-- `app_access_key`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**.
-![Screenshot of the console with accesskey section](./images/accesskey-console.png?raw=true)
 
-The parameters **\*\*kwargs** can be the following:
-- `mqtt_address`: **string**  this  is the address of the handler to which the application was registered.
-- `discovery_address`: **string**  this is the address of the discovery server to use in order to find back the address of the MQTT handler.
-The constructor returns an **MQTTClient object** set up with the application informations, ready to be connected to The Things Network.
+* `app_id`: **string** this the name given to the application when it was created.
+  ![Screenshot of the console with app section](./images/app-console.png?raw=true)
+* `app_access_key`: **string** this can be found at the bottom of the application page under **ACCESS KEYS**.
+  ![Screenshot of the console with accesskey section](./images/accesskey-console.png?raw=true)
+* `mqtt_address`: **string** this is the address of the handler to which the application was registered, in the `{hostname}:{port}` format.
```

Comment:
Please describe, in a straightforward way, the different workflows if `mqtt_address` and/or `discovery_address` are set.